storm-core/src/jvm/backtype/storm/spout/RawScheme.java

```diff
@@ -18,11 +18,13 @@
 package backtype.storm.spout;
 
 import backtype.storm.tuple.Fields;
+
+import java.nio.ByteBuffer;
 import java.util.List;
 import static backtype.storm.utils.Utils.tuple;
 
 public class RawScheme implements Scheme {
-    public List<Object> deserialize(byte[] ser) {
+    public List<Object> deserialize(ByteBuffer ser) {
```

Comment:
@arunmahadevan it might but I don't seen any one using this apart from KafkaSpout